examples/ecmascript/es5.ohm

```diff
@@ -273,15 +273,12 @@ ES5 {
                     | "(" Expression ")"  -- parenExpr
 
   // @returns an ArrayExpr AST
-  ArrayLiteral = "[" AssignmentExpressionOrElisionComma* "]"                       -- trailingComma
-               | "[" AssignmentExpressionOrElisionComma* AssignmentExpression "]"  -- noTrailingComma
-  AssignmentExpressionOrElisionComma = (AssignmentExpression | Elision) ","
-  Elision =
+  ArrayLiteral = "[" ListOf<AssignmentExpressionOrElision, ","> "]"
+  AssignmentExpressionOrElision = AssignmentExpression
+                                |                       -- elision
 
   // @returns an ObjectExpr AST
-  ObjectLiteral = "{" PropertyAssignmentComma* "}" -- trailingComma
-                | "{" PropertyAssignmentComma* PropertyAssignment "}" -- noTrailingComma
-  PropertyAssignmentComma = PropertyAssignment ","
+  ObjectLiteral = "{" ListOf<PropertyAssignment, ","> ","? "}"
```

Comment:
This doesn't have the same meaning as the original b/c it would happily accept `{,}` as an object literal. Here's a version that does:

```
ObjectLiteral
  = "{" ListOf<PropertyAssignment, ","> "}"  -- noTrailingComma
  | "{" NonemptyListOf<PropertyAssignment, ","> "," "}"  -- trailingComma
```

Unfortunately it's not as beautiful, but that's JavaScript for you :/